src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/entities/enums/User.java

```diff
@@ -1,4 +1,4 @@
-package com.softserveinc.ita.dkrutenko.pageobjects.softserve.repo;
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models.entities.enums;
```

Comment:
I mean, that enum classes, model classes, and entity classes should be placed at 'models' package

User class is model class